src/test/regress/sql/multi_materialized_view.sql

```diff
@@ -0,0 +1,90 @@
+---
+--- MULTI_MATERIALIZED_VIEW
+---
+
+-- This file contains test cases for materialized view support. It verifies
+-- various Citus features: 
+
+
+-- materialized views work
+-- insert into... select works with views
+CREATE VIEW air_shipped_lineitems AS SELECT * FROM lineitem_hash_part WHERE l_shipmode = 'AIR';
+CREATE TABLE temp_lineitem(LIKE lineitem_hash_part);
+SELECT create_distributed_table('temp_lineitem', 'l_orderkey', 'hash', 'lineitem_hash_part');
```

Comment:
I'm kind of curious about what happens if the materialized view is initially on a table that was local, but then the table is distributed.